### DISCUSSION TITLE
lockscreen display not working Android 14

### DISCUSSION THREAD

**Discussion by nekromantik** [NightscoutFoundation/xDrip#3186] [Bug reports] [answered]
Hi all
So I switched to the latest version of the app and now the blood glucose value on the lockscreen no longer displays.
need to unlock to see it.
this is on android 14 and on pixel 7 pro phone.

**Comment by Navid200** [comment]
If you mean always on display, there are Android settings that are occasionally disabled after an update.  Just verify that all settings are as advised here:
https://navid200.github.io/xDrip/docs/Display/AOD.html

**Reply by nekromantik** [reply]
hi not always on display but the lock screen. screen that appears after turning screen on if you dont have always on display

**Reply by Navid200** [reply]
But, you have always on display.  

Please explain why always on display is not good enough for you.

**Reply by nekromantik** [reply]
it uses too much battery

**Reply by Navid200** [reply, marked as answer]
Understood.

Can you check this?
https://navid200.github.io/xDrip/docs/Lock-screen.html

**Reply by nekromantik** [reply]
tried that and it works. thanks

**Reply by Navid200** [reply]
You're welcome
